AmplifyPlugins/Auth/Tests/AuthHostApp/AuthWatchApp.entitlements

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)com.amazon.aws.amplify.swift.AuthWatchAppShared</string>
```

Comment:
It is required from what I can tell. I tried manually removing it, and modifying the tests to use the access groups without the teamID (which is the same as AppIdentifierPrefix, to clarify), and the keychain throws an access error because of it. I believe the teamID is required no matter what